src/app/components/create-edit-event-page/create-edit-event-page.component.scss

```diff
@@ -44,14 +57,63 @@
     align-items: center;
 }
 
-.description-container {
+.full-width-container {
```

Comment:
it can be named .full-width, so if someone needs it for a field later, there'll not be needed another .full-width-field class